integrations/monetate/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@segment/analytics.js-integration-monetate",
   "description": "The Monetate analytics.js integration.",
-  "version": "2.2.0",
```

Comment:
If it's not a breaking change, should we just bump by a minor version?

Reply:
Sadly is a breaking change. It changes how `product viewed` behaves :(

Reply:
Oh it doesn't just add a new handler for `productListViewed`?